networkx/algorithms/components/weakly_connected.py

```diff
@@ -83,6 +83,14 @@ def number_weakly_connected_components(G):
     NetworkXNotImplemented
         If G is undirected.
 
+    Examples
+    --------
+    >>> import networkx as nx
```

Comment:
This is done automatically for all doctests so doesn't need to be specified explicitly.

```suggestion
```